BookingsAPI/Bookings.Api.Contract/Responses/UserWithClosedConferencesResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Bookings.Api.Contract.Responses
+{
+    public class UserWithClosedConferencesResponse
+    {
+        public IList<string> Username { get; set; }
```

Comment:
Because it is a list, ideally the property should be named as UserNames

Reply:
updated the list name